cmd/kubeadm/app/cmd/config.go

```diff
@@ -228,6 +229,7 @@ func NewCmdConfigMigrate(out io.Writer) *cobra.Command {
 			locally in the CLI tool without ever touching anything in the cluster.
 			In this version of kubeadm, the following API versions are supported:
 			- %s
+			- %s
```

Comment:
?

Reply:
This gives some info about api version supported by kubeadm command migrate...